changelog/generated/v1.5.0.md

```diff
@@ -0,0 +1,37 @@
+- **Breaking change**: PROJECT config version 3-alpha has been upgraded to version 3. ([#4613](https://github.com/operator-framework/operator-sdk/pull/4613))
+- Improved `generate bundle` and `generate packagemanifests` handling and help text for use outside of a project. ([#4514](https://github.com/operator-framework/operator-sdk/pull/4514))
+- (go/v3) Pin controller-runtime to v0.7.2. ([#4626](https://github.com/operator-framework/operator-sdk/pull/4626))
+- For Ansible-based operators, change the level of the proxy skipping cache logs to debug level. ([#4511](https://github.com/operator-framework/operator-sdk/pull/4511))
+
+### Bug Fixes
+
+- For Go-based operators, fix `--force` option to recreate the files via `create api` command. More info: [#kubernetes-sigs/kubebuilder#1903](https://github.com/kubernetes-sigs/kubebuilder/pull/1903). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- For Go-based operators, fix the name of the files to be edit in the comments of `[kind]_types.go` files. More info: [#kubernetes-sigs/kubebuilder#1927](https://github.com/kubernetes-sigs/kubebuilder/pull/1927). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- Upgrade the `kube-rbac-proxy` image version from 0.5.0 to 0.8.0 to address security concerns. More info [#kubernetes-sigs/kubebuilder#1955](https://github.com/kubernetes-sigs/kubebuilder/pull/1955). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- For Go-based operators, fix the endpoint names. More info [#kubernetes-sigs/kubebuilder#1910](https://github.com/kubernetes-sigs/kubebuilder/pull/1910). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- Allows initialize the projects with directories with `.`(dot) prefix names. More info [#kubernetes-sigs/kubebuilder#1944](https://github.com/kubernetes-sigs/kubebuilder/pull/1944). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- Properly consider all Go files when generating a CSV's `spec.customresourcedefinitions.owned`. ([#4507](https://github.com/operator-framework/operator-sdk/pull/4507))
+- Fixed samples kustomization.yaml generation on `operator-sdk init --plugins=helm --helm-chart=<chart>`, caused by out-of-order operations in plugin code. ([#4584](https://github.com/operator-framework/operator-sdk/pull/4584))
+- Skip CSV definitions parsing in `generate kustomize manifests` if the APIs dir does not exist, as projects may use only required APIs. ([#4624](https://github.com/operator-framework/operator-sdk/pull/4624))
+- Bump cryptography dependency in ansible-operator to 3.3.2. This will fix a potential security bug related to update() calls. Ref: https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#332---2021-02-07. ([#4539](https://github.com/operator-framework/operator-sdk/pull/4539))
```

Comment:
```suggestion
- Bump cryptography dependency in ansible-operator to `3.3.2`. This will fix a potential security bug related to update() calls. Ref: https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#332---2021-02-07. ([#4539](https://github.com/operator-framework/operator-sdk/pull/4539))
```